src/Razor/src/Microsoft.CodeAnalysis.Razor.Workspaces/TagHelperResolver.cs

```diff
@@ -16,7 +16,7 @@ internal abstract class TagHelperResolver : IWorkspaceService
     {
         public abstract Task<TagHelperResolutionResult> GetTagHelpersAsync(Project workspaceProject, ProjectSnapshot projectSnapshot, CancellationToken cancellationToken = default);
 
-        protected virtual async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project workspaceProject, RazorProjectEngine engine)
+        protected virtual async Task<TagHelperResolutionResult> GetTagHelpersAsync(Project workspaceProject, RazorProjectEngine engine, CancellationToken cancellationToken)
```

Comment:
We take a `cancellationToken` but don't use it anywhere yet, is the expectation that we'll add it to `workspaceProject.GetCompilationAsync()` later?

Reply:
Have faith in yourself 😉. Super legit issue. I need to pass it into the GetCompilationAsync method. Also your other cancellationToken comment was legit too. Shouldn't have been passing None.